cmd/protoc-gen-go-tableau-loader/messager.go

```diff
@@ -359,7 +368,7 @@ func genOrderedMapTypeDef(depth int, keys []helper.MapKey, messagerName string,
 	return ""
 }
 
-func genOrderedMapLoader(depth int, keys []helper.MapKey, messagerName string, file *protogen.File, g *protogen.GeneratedFile, message *protogen.Message, lastOrderedMapValue string) {
+func genOrderedMapLoader(depth int, keys []helper.MapKey, messagerName string, file *protogen.File, g *protogen.GeneratedFile, message *protogen.Message, lastOrderedMapValue string, needConvertBool bool) {
```

Comment:
There is no need to add parameter: `needConvertBool bool`

Reply:
changed